modules/programs/vim.nix

```diff
@@ -65,6 +65,18 @@ in {
     programs.vim = {
       enable = mkEnableOption "Vim";
 
+      source_package = mkOption {
```

Comment:
Options should as much as possible use camel cased names.
```suggestion
      sourcePackage = mkOption {
```

Reply:
Right, will change that.

Reply:
Changed.